test/vars.go

```diff
@@ -0,0 +1,66 @@
+package integration
+
+import (
+	"fmt"
+	"os"
+	"path"
+)
+
+const varsFilename = "config.tfvars"
```

Comment:
using tfvars is a workaround to assign undefined variables without causing terraform to panic

https://github.com/hashicorp/terraform/issues/19898